bin/plugin/commands/release.js

```diff
@@ -71,34 +71,64 @@ async function runReleaseBranchCreationStep(
 ) {
 	let version, releaseBranch, versionLabel;
 	await runStep( 'Creating the release branch', abortMessage, async () => {
+		let parsedMajorVersion;
+
+		const getComponentsOfRC = ( ver ) => ver.match( /(.+-rc\.)([0-9]+)/ );
```

Comment:
I think we can use `semver.parse()` instead, and inspect the resulting `prerelease` attribute. Something like

```js
const packageVersion = semver.parse( packageJson.version );
const { prerelease } = packageVersion;
```

Reply:
Example:

```
const version = semver.parse( '9.1.2-rc.1' );
console.log( version.prerelease );
```

gives `[ 'rc', 1 ]` :tada:

Reply:
Ah, good one! I'll change 👍🏻